eth/api_backend.go

```diff
@@ -428,3 +455,8 @@ func (b *EthAPIBackend) StateAtBlock(ctx context.Context, block *types.Block, re
 func (b *EthAPIBackend) StateAtTransaction(ctx context.Context, block *types.Block, txIndex int, reexec uint64) (*types.Transaction, vm.BlockContext, *state.StateDB, tracers.StateReleaseFunc, error) {
 	return b.eth.stateAtTransaction(ctx, block, txIndex, reexec)
 }
+
+type prunedHistoryError struct{}
```

Comment:
It's indeed weird to have duplicated error definitions in different package. Nitpick :)

Reply:
Yep agree I now made the definition in api_backend public instead.